clients/client-python/tests/integration/hdfs_container.py

```diff
@@ -0,0 +1,143 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import asyncio
+import logging
+import os
+import time
+
+import docker
+from docker import types as tp
+from docker.errors import NotFound
+
+from gravitino.exceptions.gravitino_runtime_exception import GravitinoRuntimeException
+
+logger = logging.getLogger(__name__)
+
+
+async def check_hdfs_status(hive_container):
```

Comment:
Can we rename "hdfs" to "hive" in this file, so later on when we implement tabular APIs, we can leverage this class to initialize hive container.

Reply:
Currently, there is no logic to check whether Hive is ready, so I am afraid that using Hive Container may cause misunderstanding to others. My suggestion is that if we need to support Hive integration tests in Python in the future, someone can refactor this and add Hive status check logic.